src/plugins/quickdump/README.md

```diff
@@ -0,0 +1,114 @@
+- infos = Information about the quickdump plugin is in keys below
+- infos/author = Klemens Böswirth <k.boeswirth+git@gmail.com>
+- infos/licence = BSD
+- infos/needs =
+- infos/provides = storage/quickdump
+- infos/recommends =
+- infos/placements = getstorage setstorage
+- infos/status = tested nodep libc compatible preview
+- infos/metadata =
+- infos/description = much quicker version of dump
+
+## Introduction
+
+`quickdump` is a storage plugin based on the `dump` format. It is a lot quicker than the old `dump` plugin, because it does not use commands
```

Comment:
How much quicker?

Reply:
I added some benchmarks.

I think in these benchmarks you can also see, that reusing the buffer would only have a minor impact. In the get only benchmark the `dump` plugin slowly catches up to the speed of the `quickdump` plugin. Most likely this is due too the fact that `quickdump` does a lot more malloc/free. Other than that I don't really see a reason why the factor should change significantly with the size of the keyset (for get only, set is another thing).